src/main/java/com/github/onsdigital/babbage/api/endpoint/search/Search.java

```diff
@@ -35,8 +41,15 @@ public void get(HttpServletRequest request, HttpServletResponse response) throws
         if (extractPage(request) == 1 && isEmpty(filter)) {
             searchAdditionalContent = true;
         }
-        search(request, getClass().getSimpleName(), searchTerm, queries(request, searchTerm, searchAdditionalContent), searchAdditionalContent)
-                .apply(request, response);
+        if (searchTerm != null) {
+            String timeSeriesRedirect = getRedirectWhenTimeSeries(request, searchTerm);
+            if (timeSeriesRedirect != null) {
+                new BabbageRedirectResponse(timeSeriesRedirect, appConfig().babbage().getSearchResponseCacheTime())
+                        .apply(request, response);
+            }
+        }
+        Map<String, SearchResult> results = search(searchTerm, queries(request, searchTerm, searchAdditionalContent), searchAdditionalContent);
+        SearchRendering.buildResponse(request, getClass().getSimpleName(), results).apply(request, response);
```

Comment:
NOt 100% sure I know what this code will do - but haven't we already built the response? line 47?

Reply:
Good spot, I'll rewrite this bit.